aroma/tests/test_integration.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+import os
+import pandas as pd
+import subprocess
+from os.path import join, split, isfile
+from argparse import Namespace
+
+from aroma import main
+
+import pytest
+
+
+def test_integration(skip_integration, nilearn_data):
+    if skip_integration:
+        pytest.skip('Skipping integration test')
+
+    test_path, _ = split(nilearn_data.func[0])
+
+    out_path = join(test_path, 'out')
+
+    confounds = pd.read_csv(nilearn_data.confounds[0], sep='\t')
+
+    mc = confounds.iloc[:, :6]
+    mc_path = join(test_path, 'mc.tsv')
+    mc.to_csv(mc_path, sep='\t', index=False, header=None)
```

Comment:
For the life of me, I _cannot_ find info on the order and units of MCFLIRT motion parameters (which AROMA assumes). Do you know what they are? Depending on the order, the translation and rotation columns may need to be switched in the fMRIPrep confounds, and depending on the units, the rotation columns may need to be converted to degrees or radians.

I'm trying to figure this out right now for #9.

Reply:
All mentions I've seen online have rotation first, but I haven't found an official source that states the proper order.

Reply:
Do you know if the rotation data are degrees or radians?

Reply:
Awesome. That helps! fMRIPrep rotations are in radians too, so we're good there.

Reply:
Based on the different order, I'd recommend doing `mc = df[["rot_x", "rot_y", "rot_z", "trans_x", "trans_y", "trans_z"]].values` instead of `df.iloc[:, :6]` (since that would put translations before rotations).

Reply:
Done in the latest commit. The motion ICs vary though and I cannot check for an exact match now.